### PR TITLE
Support bisecting Bazel to find which Bazel change breaks your build

### PR DESCRIPTION
**Pull request by meteorcloudy**
- Use --bisect=<BASE>..<HEAD> to specify the bisecting range, Bazelisk
  uses the GitHub API to get the list of commits to bisect. You may need
  to set `BAZELISK_GITHUB_TOKEN` to get around GitHub rate limit.
- BAZELISK_SHUTDOWN, BAZELISK_CLEAN can be used to run `bazel shutdown`
  or `bazel clean --expunge` between builds.

### CODE REVIEW

File: core/core.go [hunk 1]
```diff
@@ -140,14 +125,25 @@ func RunBazeliskWithArgsFunc(argsFunc ArgsFunc, repos *Repositories) (int, error
 		if err != nil {
 			return -1, fmt.Errorf("could not get the list of incompatible flags: %v", err)
 		}
-
 		if args[0] == "--migrate" {
 			migrate(bazelPath, args[1:], newFlags)
-		} else {
+		} else if args[0] == "--migrate" {
```
Comment (fweikert) [inline core/core.go:130]: you mean --strict?

Comment (fweikert) [reply core/core.go:130]: The else alone is still fine, given the outer if

Comment (meteorcloudy) [reply core/core.go:130]: Oh, right, or I should just remove this check.

Comment (meteorcloudy) [reply core/core.go]: Done

**Review by fweikert — APPROVED** [review]

**Comment by Wyverald** [comment]
nice! :D

**Pull request merged**